app/models/standing_line_item.rb

```diff
@@ -0,0 +1,16 @@
+class StandingLineItem < ActiveRecord::Base
+  belongs_to :standing_order, inverse_of: :standing_line_items
+  belongs_to :variant, class_name: 'Spree::Variant'
+
+  validates :standing_order, presence: true
+  validates :variant, presence: true
+  validates :quantity, { presence: true, numericality: { only_integer: true } }
+
+  def available_from?(shop, schedule)
+    Spree::Variant.joins(exchanges: { order_cycle: :schedules})
+    .where(id: variant_id, schedules: { id: schedule}, exchanges: { incoming: false, receiver_id: shop })
```

Comment:
Spacing around hashes look a bit unbalanced on these lines. We should probably decide on some conventions for syntax in general and set up Rubocop as part of the build pipeline to enforce it.